doc/cephfs/fuse.rst

```diff
@@ -1,53 +1,91 @@
-=======================
-Mount CephFS using FUSE
-=======================
+========================
+ Mount CephFS using FUSE
+========================
 
-Before mounting Ceph via "Filesystem in USErspace" (FUSE), ensure that the client
-host has a copy of the Ceph configuration file and a keyring with CAPS for the
-Ceph metadata server.
+Prerequisite
+------------
 
-#. From your client host, copy the Ceph configuration file from the monitor host 
-   to the ``/etc/ceph`` directory. :: 
+Before mounting CephFS, ensure that the client host (where CephFS has to be
+mounted and used) has a copy of the Ceph configuration file (i.e.
+``ceph.conf``) and a keyring of the CephX user that has CAPS for the Ceph MDS.
+Both of these files must be present on the host where the Ceph MON resides.
 
-	sudo mkdir -p /etc/ceph
-	sudo scp {user}@{server-machine}:/etc/ceph/ceph.conf /etc/ceph/ceph.conf
+#. Copy the Ceph conf from the MON host to client host at ``/etc/ceph``::
 
-#. From your client host, copy the Ceph keyring from the monitor host to 
-   to the ``/etc/ceph`` directory. :: 
+    # on client host
+    mkdir -p /etc/ceph
+    ssh {user}@{mon-host} "sudo ceph config generate-minimal-config" | sudo tee /etc/ceph/ceph.conf
 
-	sudo scp {user}@{server-machine}:/etc/ceph/ceph.keyring /etc/ceph/ceph.keyring
+#. Ensure that the conf has appropriate permissions::
 
-#. Ensure that the Ceph configuration file and the keyring have appropriate 
-   permissions set on your client machine  (e.g., ``chmod 644``).
+    chmod 644 /etc/ceph/ceph.conf
 
-For additional details on ``cephx`` configuration, see 
-`CEPHX Config Reference`_.
+#. Copy the Ceph keyring from the MON host to client host at ``/etc/ceph``::
 
-To mount the Ceph file system as a FUSE, you may use the ``ceph-fuse`` command.
-For example::
+    scp {user}@{server-machine}:/etc/ceph/ceph.keyring /etc/ceph/ceph.keyring
 
-	sudo mkdir /home/username/cephfs
-	sudo ceph-fuse -m 192.168.0.1:6789 /home/username/cephfs
+#. Ensure that the keyring has appropriate permissions::
 
-If you have more than one file system, specify which one to mount using
-the ``--client_mds_namespace`` command line argument, or add a
-``client_mds_namespace`` setting to your ``ceph.conf``.
+    chmod 600 /etc/ceph/ceph.keyring
 
```

Comment:
A separate client setup page would be nice. But do we have more content for a separate page? Just 4 points sounds pretty short for a page.

Reply:
I was thinking something like a "Client bootstrap" page that walks someone through fetching a ceph.conf file and keyring and installing them in the correct place for their distro. Then we could just have the FUSE, kcephfs, and rbd pages point to that as a prerequisite.

We don't have to do it in this PR, but I think it'd cut down some duplication and make it easier to deal with if those procedures change in the future.

Reply:
> We don't have to do it in this PR, but I think it'd cut down some duplication and make it easier to deal with if those procedures change in the future.

Ok. I'll not mark this conversation resolved so that I can get this done after this PR meges.

Reply:
Let's call this resolved. I've opened https://tracker.ceph.com/issues/42447 to track creating a consolidated client setup page.